lib/beacon_web/live/page_live.ex

```diff
@@ -24,7 +24,11 @@ defmodule BeaconWeb.PageLive do
   end
 
   def render(assigns) do
-    {{site, path}, {_page_id, _layout_id, format, template, _page_module, _component_module}} = lookup_route!(assigns.__site__, assigns.__live_path__)
+    {{site, path}, {_page_id, _layout_id, format, templates, _page_module, _component_module}} =
+      lookup_route!(assigns.__site__, assigns.__live_path__)
+
+    template = choose_template(templates)
```

Comment:
How do you see the UI for choosing the weights? If users choose A/40 and B/60 then will it display 0 for primary? Similarly if they choose A/10 and B/20 would it display 70 for primary?

Reply:
Yes I think on a "Variants" page in `/admin` it would list each variant with an input for the weight.  We can include validation where `if sum(weights) > 100` then an error is displayed and changes cannot be submitted.  Then at the bottom (or top) of the variants page, it can show `Primary template weight` with live updates based on `100 - sum(weights)`.  Basically the same as in your example 🙂

Reply:
Sounds good 👍🏻